recipe/meta.yaml

```
@@ -9,55 +9,56 @@ source:
   sha256: 33612eb1528a55ce4700f0c420c7775f4ff0533d5f256251ad6c07a17787f774
 
 build:
-  noarch: python
+  skip: true  # [s390x]
   number: 0
-  script: {{ PYTHON }} -m pip install . --no-deps -vv
+  script: {{ PYTHON }} -m pip install --no-deps --no-build-isolation -vv .
 
 requirements:
   host:
     - pip
-    - python >=3.8.*
+    - python
+    - wheel
```

```suggestion
    - wheel
    - setuptools
```
See https://github.com/alteryx/featuretools/blob/1498c9a9723deff9b049e4bb70a4859c70331942/pyproject.toml#L208-L213